docs/java/java-debugging.md

```diff
@@ -1,15 +1,15 @@
 ---
 Order: 3
 Area: java
-TOCTitle: Debugging
+TOCTitle: Running and Debugging
 ContentId: 929e5410-3bfe-4107-b331-565afe5d341f
-PageTitle: Java Debug in Visual Studio Code
+PageTitle: Run and Debug Java in Visual Studio Code
 DateApproved: 1/2/2019
-MetaDescription: See how you can debug your Java code locally, and in the cloud.
+MetaDescription: See how you can run and debug your Java code locally, and in the cloud.
 MetaSocialImage:
 ---
 
-# Debugging Java in Visual Studio Code
+# Running and Debugging Java in Visual Studio Code
```

Comment:
Are you sure this is neccesary? This will cause the topic title to wrap which doesn't look very nice.

Reply:
I see. If that's too long, can we shorten it to run and debug? Lots of students don't really understand debug but they do want to run their Java code.